pom.xml

```diff
@@ -27,24 +27,26 @@
     <EMISSARY_VERSION>${project.version}</EMISSARY_VERSION>
     <argLine />
     <checkstyleFormatter>${project.basedir}/contrib/checkstyle.xml</checkstyleFormatter>
+    <dep.bcprov-jdk15on.version>1.64</dep.bcprov-jdk15on.version>
     <dep.commons-cli.version>1.5.0</dep.commons-cli.version>
     <dep.commons-codec.version>1.15</dep.commons-codec.version>
     <dep.commons-collections.version>4.4</dep.commons-collections.version>
+    <dep.commons-compress.version>1.20</dep.commons-compress.version>
     <dep.commons-exec.version>1.3</dep.commons-exec.version>
     <dep.commons-io.version>2.11.0</dep.commons-io.version>
     <dep.commons-lang.version>3.12.0</dep.commons-lang.version>
     <dep.commons-pool2.version>2.11.1</dep.commons-pool2.version>
     <dep.dropwizard.metrics.version>4.2.4</dep.dropwizard.metrics.version>
     <dep.errorprone.javac.version>9+181-r4173-1</dep.errorprone.javac.version>
     <dep.errorprone.version>2.10.0</dep.errorprone.version>
+    <dep.gson.version>2.8.6</dep.gson.version>
     <dep.guava.version>31.1-jre</dep.guava.version>
     <dep.httpclient.version>4.5.13</dep.httpclient.version>
     <dep.httpcore.version>4.4.15</dep.httpcore.version>
     <dep.impsort.version>1.6.2</dep.impsort.version>
     <dep.jackson.version>2.13.4</dep.jackson.version>
-    <!-- jersey and tika provide; converge on latest -->
+    <dep.jakarta.activation-api.version>1.2.2</dep.jakarta.activation-api.version>
```

Comment:
What was the reason you needed to specify jakarta.activation? 
org.glassfish.jersey.media:jersey-media-json-jackson:jar:2.35:compile brings it in already but is the 1.2.1 version

Reply:
when I deleted the the tika dependency, I got a version conflict on jakarta.activation with 2 other dependencies. Rather than exclude it from one of the  dependencies, I added the newest version as a dependency (this approach was based on  previous feedback in another PR).